.github/workflows/start-release.yml

```diff
@@ -16,6 +16,7 @@ on:
         default: "feature"
         type: "choice"
         options:
+          - "major"
```

Comment:
Where is the logic that understands the "major" means increment the first number in x.y.z?
the "bumpVersionFragment" is a special builtin of GHA?

Reply:
This logic is understood by the third-party workflow we use to increment the version - it accepts either `major`, `feature` or `bug` as inputs and increments the appropriate semver number.

Reply:
Can you reference this third-party workflow in a comment before the options, so people can understand the magic?
I still don't know what is this third-party workflow.

Reply:
Yep, will do.